Sources/DiagnosticsLogger.swift

```diff
@@ -137,7 +141,7 @@ extension DiagnosticsLogger {
         queue.async { [unowned self] in
```

Comment:
<!--
  0 failure: 
  1 warning:  It is safer to us...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DiagnosticsLogger.swift;
  Line: 141;
-->

- :warning: It is safer to use weak instead of unowned